runner/host/iptables.go

```diff
@@ -38,12 +38,21 @@ func (r IPTables) ID() string {
 
 func (r IPTables) Run() op.Op {
 	startTime := time.Now()
-	if r.OS == "linux" {
+	if r.OS != "linux" {
```

Comment:
Unrelated to the change at hand, but this looked like a little bug that had crept in as part of a recent merge.